docs/pages/contracts/v4/resources/addresses.mdx

```diff
@@ -1,7 +1,29 @@
 import { Callout } from 'vocs/components'
 
-# Addresses (Work in progress)
+# Addresses 
 
 <Callout type="note">
-  Stay tuned! page will be updated when contracts are deployed at Sepolia!
-</Callout>
\ No newline at end of file
+  PCS v4 have been deployed on Sepolia
+</Callout>
+
+--- 
+
+From https://github.com/pancakeswap/pancake-v4-core
+
+| Contract | Address  |
+| ------ | ------ |
+| Vault | 0x42A228A7fB040033Dc49FdD5ed909F2cA742271D |
+| CLPoolManager | 0xC6A2Db661D5a5690172d8eB0a7DEA2d3008665A3 |
+| BinPoolManager | https://github.com/pancakeswap/pancake-v4-hooks |
+
```

Comment:
why for BinPoolManager this is not a address ?

Reply:
my bad, updated